### ISSUE TITLE
[Bug][a11y] The scale from Accessibility is not read by screen readers

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#16392]
## Steps to reproduce 
1. Have the Accessibility scanner app installed and opened.
2. Go to Settings - Accessibility and scan the page.

### Expected behavior
No suggestions are made.

### Actual behavior
The scale doesn't have a label readable by screen readers.
Turning voice assistant on, indeed, the scale is not read. The assistant only announces 100% but not "scale" or "seek bar".

### Device information
* Android device: Google Pixel (Android 10)
* Fenix version: 11/3 Nightly
 
<img src="https://user-images.githubusercontent.com/40694158/98228395-34058d80-1f61-11eb-88d4-7fe2b47ca9ff.png" width="300" height="550" />

**Comment by mcarare** [comment]
The scanner result has nothing to do with announcing the element type. In this case, no label should be needed if the seek bar is announced as such by Talkback. Tested and it announces "seek control" on multiple devices.
Can QA please retest this? TY!

**Comment by softvision-miralobontiu** [comment]
I've verified on three different devices: Samsung Galaxy Tab S6 (Android 10), Sony Xperia Z5 Premium (Android 7.1.1), and OnePlus 5T (Android 9). 
On the OnePlus device, the voice assistant reads the "seek control", on the other two only announces "x % slider" (x being the percentage on which the scale is set to).
@mcarare should I close this issue?

**Comment by mcarare** [comment]
@softvision-miralobontiu Yes, this can be closed, given the behavior described in the comment above. Thank you!
